Dockerfile.windows

```diff
@@ -0,0 +1,25 @@
+ARG BASE_IMAGE=scracth
```

Comment:
We might even remove the default `scratch` value?

Reply:
Done!